core/spec.md

```diff
@@ -2677,11 +2685,16 @@ The following describes the attributes of Registry model:
     it is valid for an implementation to only support one (`1`) Version when
     `maxversions` is set to `0`.
   - When the limit is exceeded, implementations MUST prune Versions by
-    deleting the oldest Version (based on creation times) first, skipping the
-    Version marked as "default". An exception to this pruning rule is if
-    `maxversions` value is one (`1`) then the newest Version of the Resource
-    MUST always be the "default" and the `setdefaultversionsticky` aspect
-    MUST be `false`.
+    deleting the oldest Version first, skipping the
+    Version marked as "default". The strategy for finding the oldest Version
+    to delete is as follows:
+    1. Find all Versions that do not have an existing `ancestor`
+    1. If many exist, find all the Version with the oldest `created` date
+    1. If many exist, find the first Version according to alphabetical order
+       of the `versionid`-attribute and delete it
```

Comment:
s/-/ /
I'd stop after "attribute".
Then on the next line, outside the numbered list say: Once the single oldest Version is determined, delete it.